uSync.Migrations/Migrators/BlockGrid/GridToBlockGridMigrator.cs

```diff
@@ -112,6 +118,13 @@ private ILegacyGridEditorsConfig GetGridConfig(SyncMigrationContext context)
 			return string.Empty;
 		}
 
+		var dataTypeAlias = context.ContentTypes.GetDataTypeAlias(contentProperty.ContentTypeAlias, contentProperty.PropertyAlias);
```

Comment:
Hi @bielu & @KevinJump , apologies for the delay in testing this.

I've had a look through and it seems that the acquisition of the `dataTypeAlias` doesnt seem to work. Im not 100% sure what `getDataTypeAlias()` is fully meant to do, but in my case all the values come through as empty strings, therefore not retrieving what is expected. I have however done a little re-work and got it to work with the following code (may need refining, but again, I dont know if `GetDataTypeAlias` is meant to be a shortcut to doing this):

```suggestion
		var dataTypeAlias = "";

		var dataTypeGuid = context.ContentTypes.GetEditorAliasByTypeAndProperty(contentProperty.ContentTypeAlias, contentProperty.PropertyAlias)?.DataTypeDefinition ?? Guid.Empty;

		if (dataTypeGuid != Guid.Empty)
        {
            dataTypeAlias = context.DataTypes.GetByDefinition(dataTypeGuid)?.DataTypeName;
        }
```

Reply:
Thanks for comments will check them tomorrow :)